app/Http/Controllers/SettingsController.php

```diff
@@ -768,6 +768,12 @@ public function postLabels(Request $request)
         } else {
             $setting->labels_display_tag = 0;
         }
+        
+         if (Input::has('labels_display_company_name')) {
+            $setting->labels_display_company_name = 1;
+        } else {
+            $setting->labels_display_company_name = 0;
+        }
 
```

Comment:
A few small (again, nitpicky) things here:

1) We've been switching over to the more modern `$request->input('foo')` (or in this case, `$request->has('foo')`) notation (instead of the older `Input::get()` facade) for request data (and this method already passes it as part of the signature)

2) For something small like this, I usually like to keep the annotation simpler, like:

```
$setting->labels_display_company_name = $request->input('labels_display_company_name', '0');
```

We don't really need all the if/else business, we just need to tell the code to take the value if there is one submitted, and if not set the value to 0.